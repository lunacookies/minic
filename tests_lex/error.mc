abc$efg@ijk`mno?qrs
