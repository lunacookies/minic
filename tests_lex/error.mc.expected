{
	IDENTIFIER 0..3
	ERROR 3..4
	IDENTIFIER 4..7
	ERROR 7..8
	IDENTIFIER 8..11
	ERROR 11..12
	IDENTIFIER 12..15
	ERROR 15..16
	IDENTIFIER 16..19
}
tests_lex:1:4: error: invalid token “$”
tests_lex:1:8: error: invalid token “@”
tests_lex:1:12: error: invalid token “`”
tests_lex:1:16: error: invalid token “?”
